.travis.yml

```diff
@@ -8,11 +8,12 @@ matrix:
     - python: 3.6
     - python: 3.4.1
 
-script:  # Handles creating a blank config file so imports dont fail
-  - mkdir -p instance
-  - touch instance/config.py
+env:
+  - APP_CONFIG_NAME=test
+
+script:
   - pytest
-  - flake8 middleware/ --ignore=F403,F405
+  - flake8 middleware/ --ignore=F401,F403,F405
```

Comment:
Good question. Is this error raised when we import * from a module? If not, we should try removing and see if we get any errors that we can't fix.